transactions.py

```diff
@@ -1,37 +1,73 @@
 from create_db_tables import IP_ADDRESSES, KEYSPACES
 from schema import *
 
+import sys
+
 from cassandra import ConsistencyLevel
 from cassandra.cqlengine import connection
 
 def main():
     connection.setup(IP_ADDRESSES, KEYSPACES[0])
+    first_line = sys.stdin.readline().split(',')
```

Comment:
Feel free to just pass in the actual arguments here in main() if it makes testing easier